apis/python/src/tiledbsoma/experimental/ingest.py

```diff
@@ -607,31 +796,70 @@ def _write_visium_spots(
     return soma_point_cloud
 
 
-def _write_visium_images(
-    image_pyramid: MultiscaleImage,
-    scale_factors: Dict[str, Any],
+def _create_or_open_scene(
+    uri: str,
     *,
-    input_hires: Union[None, str, Path],
-    input_lowres: Union[None, str, Path],
-    input_fullres: Union[None, str, Path],
     ingestion_params: IngestionParams,
+    context: Optional["SOMATileDBContext"],
+    additional_metadata: "AdditionalMetadata" = None,
+) -> Scene:
+    try:
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+            scene = Scene.create(uri, context=context)
+    except (AlreadyExistsError, NotCreateableError):
+        # It already exists. Are we resuming?
+        if ingestion_params.error_if_already_exists:
+            raise SOMAError(f"{uri} already exists")
+        scene = Scene.open(uri, "w", context=context)
+
+    add_metadata(scene, additional_metadata)
+    return scene
+
+
+def _create_visium_tissue_images(
```

Comment:
A comment or docstring indicating the MultiscaleImage is returned open for writing might be helpful.

Reply:
I'll add updated docstrings on the merge to `main`.